### PR TITLE
add PATENTS and OpenFontLicense

### LINKED ISSUE #38: Add proper licensing to everything
- PATENTS and BSD
 - Font licenses
 - Icon/image licenses
 - Credits for everything

### PR DESCRIPTION
**Pull request by maryx**
Resolves #38.
Adds PATENTS and OpenFontLicense.txt for using Raleway. The project already has a LICENSE file.
Please let me know if I'm missing any licenses. Also, should each Flutter App require a license (e.g. each lib directory), or only the root repo (udacity-course)?

Thanks!

### CODE REVIEW

File: course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Flutter Udacity Course Unit Converter App
+
+// Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (Hixie) [inline course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE]: no need for the slashes in this file

File: course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+--------------------------------------------------------------------------------
```
Comment (Hixie) [inline course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE]: no need for the separator at the end, only between licenses

File: todo/LICENSE [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
```
Comment (Hixie) [inline todo/LICENSE]: extraneous blank line

File: course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE (hunk 1 above)
Comment (maryx) [reply course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE]: Done.

File: course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE (hunk 2 above)
Comment (maryx) [reply course/01_hello_rectangle/solution_01_hello_rectangle/LICENSE]: Done.

File: todo/LICENSE (hunk 3 above)
Comment (maryx) [reply todo/LICENSE]: Done.

**Comment by maryx** [comment]
Updated licenses for apps and exercises 1-5. Also added updated licenses for exercises 6-10.

**Comment by Hixie** [comment]
LGTM

**Pull request merged**